### ISSUE TITLE
Probably one typo

### ISSUE THREAD

**Issue by peiwang062** [MadryLab/robustness#30]
line 177 of attacker.py probably should be                     

bloss = loss.clone().detach()

**Comment by dtsip** [comment]
You are right, thanks for catching this! Fixed in 7c1e88e29ed68e796e8308e203b39c5ab78dc144

This does not affect anything since `loss` was always `losses` when the function was used.
